### DISCUSSION TITLE
Add a file to ~/Library

### DISCUSSION THREAD

**Discussion by MattHardcastle** [Homebrew/discussions#1616] [Tap maintenance and Homebrew development] [answered]
I’m writing a Formula for a native component of a Firefox plugin. In order to complete the installation of the component I need to add a manifest file to Firefox’s directory in `~/Library/Application Support`. 

I'v ran into a couple issues. The HOMEDIR environment variable is overwritten in the Formula so I can't use it to location Firefox's support directory. When I hard-coded the directory for my machine I ran into permissions issues writing to the directory.

Is adding files into `~/Library` supported in Homebrew?

**Comment by carlocab** [comment, marked as answer]
It's not, I'm afraid. You can only write into the formula's prefix and the build directory at build time and (I think) only `HOMEBREW_PREFIX` at install time.
